### PR TITLE
Reorganize Library Overview layout by unifying sections

### PR DESCRIPTION
**Pull request by jorgepiloto**
Partially solves for #62.

- [x] **Add references to ansys-templates**

### CODE REVIEW

File: doc/source/library_description/template.rst [hunk 1]
```diff
@@ -2,21 +2,88 @@
 Library Template
 ################
 
-Within the `PyAnsys Organization <https://github.com/pyansys>`_, there
-is a `template repository <https://github.com/pyansys/template>`_
-where you can `create a repository from a template`_ and create a 
-PyAnsys project that follows the guidelines specified in this developer's
-guide.
+Starting a new project from scratch is a tedious task. For this reason, the
+`ansys-tempaltes` tool was created in order to simplify the starting process and
```
Comment (MaxJPRey) [inline doc/source/library_description/template.rst:6]:
```suggestion
`ansys-templates` tool was created in order to simplify the starting process and
```

Comment (jorgepiloto) [reply doc/source/library_description/template.rst:6]: Good catch @MaxJPRey, solved in the incoming pushed changes.

Comment (akaszynski) [inline doc/source/library_description/template.rst:6]:
```suggestion
`ansys-templates` tool was created in order to simplify the starting process and
```

File: doc/source/library_description/template.rst [hunk 2]
```diff
@@ -2,21 +2,88 @@
...
+There are two templates which can be used as basis for creating new PyAnsys
+projects. These are the ``pyansys`` and the ``pyansys-advanced`` templates. 
+
+.. important::
+
+   Install `ansys-templates`_ to access these templates. Refer to the
+   `ansys-templates user guide`_ for more information on how to use this tool.
+
+
+PyAnsys template
```
Comment (akaszynski) [inline doc/source/library_description/template.rst:44]:
```suggestion
PyAnsys Template 
```

Comment (akaszynski) [reply doc/source/library_description/template.rst:44]: Some of these section titles are capitalized and some are not.

**Comment by akaszynski** [comment]
Recommend leaving the first two tasks for another PR, I've got a section to add to the guidelines.

Recommend merge.

**Review by akaszynski — APPROVED** [review]

**Comment by jorgepiloto** [comment]
No problem, merging this 🚀

**Pull request merged**